build/speaker/keepalived.tmpl

```diff
@@ -0,0 +1,33 @@
+global_defs {
+  vrrp_version 3
+  script_user root
+  enable_script_security
+  router_id {{ .name }}
+}
+
+
+#Check if the VIP list is empty
+
+{{ range $name, $instance := .instances }}
+{{ if $instance.Enabled }}
+vrrp_instance {{ $name }} {
+  state BACKUP
```

Comment:
Just one question: Why is there a fixed value BACKUP here? When will it become MASTER?

Reply:
Here's what the [official documentation](https://www.keepalived.org/manpage.html) says:
Initial state, MASTER|BACKUP If the priority is 255, then the instance will transition immediately to MASTER if state MASTER is specified; otherwise the instance will wait between 3 and 4 advert intervals before it can transition, depending on the priority.

Reply:
ok.